apsuite/orbcorr/si_calc_bump_matrix_at_bc_magnet.py

```diff
@@ -7,14 +7,16 @@
 from pymodels import si
 import pyaccel
 from . import OrbitCorr
+from . import CorrParams
 
 
-def calc_matrices(minsingval=0.2, mcidx=0, deltax=10e-6):
+def calc_matrices(minsingval=0.2, mcidx=0, deltax=10e-6, n_bpms_out=2):
```

Comment:
why do you need this extra argument? The code was developed to work without this need. Did you observe some difference in the calculated matrices?